multitenancy.md

```diff
@@ -0,0 +1,94 @@
+# Multitenancy
+
+This document contains sample instructions on how to setup one service cluster together with multiple workload clusters. As stated, the instructions below are just samples, you need update them according to your situation and requirements. You will find more details on the commands executed in the cluster and apps repos.
+
+The following repositores and commits have been tested
+- [ck8s-cluster](https://github.com/elastisys/ck8s-cluster/tree/324ac074a337b99b83a464ce0189ed2c0f267e19)
+- [compliantkubernetes-apps](https://github.com/elastisys/compliantkubernetes-apps)
+
+The cluster setup will be as follows
+- `mtt-0` will be the the repository from where the service cluster will be installed.
+- `mtt-{1,2}` will be repositories from where we'll install workload clusters.
+
+Simply do a _search and replace_ if you want to name your clusters something else.
+`mtt` - simply stands for _multi-tenancy-test_ :)
+
+### Infrastructure and kubernetes
+1. Set some variables
+  ```bash
+  domain=<your_domain>
+  pgp_fp=<your_pgp_fp>
+  config_path=<path_to_store_config_repos>
+  ```
+
+1. Init repos
+  ```bash
+  ./dist/ck8s_linux_amd64 init mtt-0 exoscale --flavor dev --pgp-fp ${pgp_fp} --config-path ${config_path}/mtt-0
+  ./dist/ck8s_linux_amd64 init mtt-1 exoscale --flavor dev --pgp-fp ${pgp_fp} --config-path ${config_path}/mtt-1
+  ./dist/ck8s_linux_amd64 init mtt-2 exoscale --flavor dev --pgp-fp ${pgp_fp} --config-path ${config_path}/mtt-2
+  ```
+
+2. For each repo edit the configuration files
+
+3. Create infrastructure and install kubernetes for the clusters
+  ```bash
+  ./dist/ck8s_linux_amd64 apply --cluster sc --config-path ${config_path}/mtt-0
+  ./dist/ck8s_linux_amd64 apply --cluster wc --config-path ${config_path}/mtt-1
+  ./dist/ck8s_linux_amd64 apply --cluster wc --config-path ${config_path}/mtt-2
+  ```
+
+3. Get ip of sc loadbalancer
+  ```bash
+  sc_lb_ip=$(./dist/ck8s_linux_amd64 internal terraform output --cluster sc --config-path ${config_path}/mtt-0 | jq -r '.sc_ingress_controller_lb_ip_address.value')
+  ```
+
+4. Create DNS records
+  ```bash
+  exo dns add A ${domain} -a ${sc_lb_ip} -n *.ops.mtt-0
+  exo dns add A ${domain} -a ${sc_lb_ip} -n *.mtt-0
```

Comment:
I don't think we have the `exo` cli as a requirement anywhere. So either add a link to where one can download it or instead use our `ck8s-dns` repo.

Reply:
ck8s-dns is not an open project. I'll add a link to exo cli

Reply:
Good point, forgot about that.